src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -132,19 +145,17 @@ void VtolAttitudeControl::vehicle_cmd_poll()
 
 	while (_vehicle_cmd_sub.update(&vehicle_command)) {
 		if (vehicle_command.command == vehicle_command_s::VEHICLE_CMD_DO_VTOL_TRANSITION) {
-			vehicle_status_s vehicle_status{};
-			_vehicle_status_sub.copy(&vehicle_status);
 
 			uint8_t result = vehicle_command_ack_s::VEHICLE_CMD_RESULT_ACCEPTED;
 
 			const int transition_command_param1 = int(vehicle_command.param1 + 0.5f);
 
 			// deny transition from MC to FW in Takeoff, Land, RTL and Orbit
 			if (transition_command_param1 == vtol_vehicle_status_s::VEHICLE_VTOL_STATE_FW &&
-			    (vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_TAKEOFF
-			     || vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_LAND
-			     || vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_RTL
-			     ||  vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_ORBIT)) {
+			    (_vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_TAKEOFF
+			     || _vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_LAND
+			     || _vehicle_status.nav_state == vehicle_status_s::NAVIGATION_STATE_AUTO_RTL
```

Comment:
@sfuhrer Do you want to keep this line ? We've removed it in our fork as we don't see why we would at all cost inhibit a transition to FW when doing an RTL. Maybe you're far away from destination and want to reach it faster.

Reply:
I look at RTL as a fully contained, autonomous flight mode. The user shouldn't be able to interact with it in any way (beside nudging the vehicle slightly on the final descend). And as there is no logic within RTL that triggers a transition to FW we can prohibit it completely in my eyes, like it's done here.
If the vehicle triggers a RTL in hover far away, but the operator still has data connection, then one is able to switch out of RTL, transition, and engage RTL again. For me this is the more predictable operation instead of allowing transitions in RTL (which pose a big safety risk if done by accident close to the ground). 
Anyway, this would be stuff for a separate PR :)